tests/gabc-gtex/bugs/fix-42-a.tex

```diff
@@ -1,10 +1,10 @@
-% File generated by gregorio 4.0.0-beta2-proof-read-a62712b-2511
-\GregorioTeXAPIVersion{4.0.0-beta2}%
-\GreBeginScore{58d6f07be381ad3e2f81429b36a877b888ba1515}{6}{5}{0}{0}{}%
+% File generated by gregorio 4.0.0-rc1-fix-650-c38d205-2619
+\GregorioTeXAPIVersion{4.0.0-rc1}%
+\GreBeginScore{44f13a3f54ece81ca04469d5a87b6481e02a7a27}{6}{5}{0}{0}{}%
 \GreSetInitial{A}%
 \GreBeginNotes %
 \GreSetInitialClef{c}{4}{3}%
-\GreSyllable{\GreSetThisSyllable{}{\GreFirstWord{}}{}{\GreFirstWord{A}}{\GreFirstWord{}}}{}{\GreFirstWord}{0}{\GreSetNextSyllable{\GreFirstWord{m}}{\GreFirstWord{e}}{\GreFirstWord{n}}{\GreFirstWord{m}}{\GreFirstWord{e}\GreFirstWord{n}}}{}{0}{}{%
+\GreSyllable{\GreSetThisSyllable{}{\GreFirstWord{}}{}{\GreFirstWord{}}}{}{\GreFirstWord}{0}{\GreSetNextSyllable{\GreFirstWord{m}}{\GreFirstWord{e}}{\GreFirstWord{n}}{\GreFirstWord{m}}{\GreFirstWord{e}\GreFirstWord{n}}}{}{0}{}{%
```

Comment:
Are you sure this is right?  You have a `\GreFirstWord` without any arguments (third one in the line).